src/plugin/peripheral/nodes/message-node.ts

```diff
@@ -51,12 +51,12 @@ export class MessageNode extends PeripheralBaseNode {
         return undefined;
     }
 
-    public performUpdate(): Thenable<boolean> {
-        return Promise.resolve(false);
+    public async performUpdate(): Promise<boolean> {
```

Comment:
Same question about `async` as above. Here and for `updateData`.

Reply:
See above.